guidelines/sample-server/files/models.json

```diff
@@ -81,5 +81,23 @@
         "format": "double"
       }
     }
+  },
+  "ErrorMessage": {
+    "description": "A shared model for error message",
+    "required" : ["error-code","reason"],
+    "properties": {
+      "error-code": {
```

Comment:
@webron who cares, it’s legal.  But because you asked…

scala> val `foo-bar` = "learn to love it”

> On May 20, 2015, at 9:05 AM, Ron <notifications@github.com> wrote:
> 
> In guidelines/sample-server/files/models.json <https://github.com/swagger-api/swagger-spec/pull/372#discussion_r30718766>:
> 
> > @@ -81,5 +81,23 @@
> >          "format": "double"
> >        }
> >      }
> > +  },
> > +  "ErrorMessage": {
> > +    "description": "A shared model for error message",
> > +    "required" : ["error-code","reason"],
> > +    "properties": {
> > +      "error-code": {
> Which languages support dashes in variable names? The common practice is to use camel case or snake case.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/swagger-api/swagger-spec/pull/372/files#r30718766>.
>